### PR TITLE
Add ZLGotoSandboxPlugin Plugin

### PR DESCRIPTION
**Pull request by MakeZL**
You can quickly enter the iOS simulator Xcode plugin!

### CODE REVIEW

**Comment by MakeZL** [comment]
Hi guys,
         This is My first plugin , you can quickly jump sandbox.
         Support Shortcut keys (Only in the running application) shift + common + w
                                                                                                                              - thanks

**Comment by jurre** [comment]
Hey @MakeZL, thanks for adding your plugin!

However, it looks like you added it as a template, and the JSON is invalid. Could you fix that?

Thanks!

**Comment by MakeZL** [comment]
Hey @jurre , Sorry. I didn't notice that. Submitted, you think it's ok

Thanks!

File: packages.json [hunk 1]
```diff
@@ -1243,6 +1243,12 @@
         "name": "PIXNETTemplate",
         "url": "https://github.com/pixnet/PIXNETTemplate",
         "description": "File Template for PIXNET coding"
+      },
+      {
+        "name": "ZLGotoSandboxPlugin",
+        "url": "https://github.com/MakeZL/ZLGotoSandboxPlugin",
+        "description": "You can quickly enter the iOS simulator Xcode plugin!",
+        "screenshot": "https://github.com/MakeZL/ZLGotoSandboxPlugin/blob/master/screenshot.png
```
Comment (jurre) [inline packages.json]: This is missing the closing "

**Comment by jurre** [comment]
I've commented inline with an error in the json syntax, but it is also still listed under templates. You'll need to move it to plugins instead in order for people to install it

**Comment by MakeZL** [comment]
Hey @jurre  , Is that ok?

**Comment by jurre** [comment]
Still listed as a template, it should be under the plugins key in the json file

**Comment by MakeZL** [comment]
@jurre ok, I modified the packages.json

**Comment by MakeZL** [comment]
Edit ZLGotoSandbox name 。

**Comment by MakeZL** [comment]
Hey , @jurre  will be able to merge my code?
- Thx

**Comment by jurre** [comment]
No, it is still under templates. It should be under plugins..

**Comment by MakeZL** [comment]
@jurre I've been on top of the template. OK?

**Comment by jurre** [comment]
Thanks

**Pull request merged**